src/fmnoise/coldbrew.clj

```diff
@@ -139,6 +149,7 @@
   (let [options (meta f)
         condition-fn (-> f meta :when)
         cache (build-cache options (when-not condition-fn f))]
+    ^{::cache cache}
```

Comment:
I feel like it's pretty much an implementation detail, so maybe we could add a helper for getting cache from the meta? 🤔

Reply:
Ah, there is a `cached` function too, so I need to add a helper for both `defcache` and `cached`🤔

Reply:
Personally, I prefer meta because:
- It exists on the same level of execution - just a part of the result
- There's no extra complexity with cache IDs, storage layers, etc.
- I was able to link it to the source caching object with minimal changes, and having access to that caching object is all I need right now.

Yes, integrating this meta into `defcached` and accessing it can be a bit tricky, but it works. And for the end user, it’s just a `cached->cache` or `defcached->cached`.

---
That said, if you have a clear idea for a different approach, I’d be really interested to see how it turns out =)